web/packages/teleport/src/UnifiedResources/ResourceCard.tsx

```diff
@@ -584,34 +595,42 @@ function PinButton({
   pinned,
   hovered,
   setPinned,
+  pinningDisabled,
 }: {
   pinned: boolean;
   hovered: boolean;
   setPinned: (id: string) => void;
+  pinningDisabled: boolean;
 }) {
   const copyAnchorEl = useRef(null);
+  const tipContent = pinningDisabled
+    ? 'Upgrade this leaf cluster to pin its resources'
```

Comment:
i think it's best to include what version to upgrade. we can grab version with:

```
  const ctx = useTeleport();
  const cluster = ctx.storeUser.state.cluster;
```

i think the language should be more like `Cluster <cluster-name> does not support pinning resources. Upgrade to <version>` or something like that

Reply:
Kenny had suggested to keep it simple with my current version but I agree that maybe a bit more detail should be included. I've changed it to  `This cluster does not support pinning resources. To enabled, upgrade to 14.1 or newer.`

This specifies the version they need (its releasing with 14.1) and that "this" cluster does not support it. I was thinking of using the cluster id but I think it would look weird having something like `teleport.zarquon.sh does not support...`

Idk, thoughts?